cypress/e2e/signIn.cy.js

```diff
@@ -2,6 +2,25 @@
 
 describe('Sign In page', () => {
   it('should provide an ability to log in', () => {
+    cy.visit('https://react-redux.realworld.io');
 
+    cy.get(':nth-child(2) > .nav-link').click();
```

Comment:
Avoid using complex selectors like ':nth-child(2) > .nav-link'. These are brittle and can break if the structure of the HTML changes. Consider adding data-test attributes to make your tests more robust.